pagespeed/system/system_rewrite_driver_factory.h

```diff
@@ -258,6 +258,18 @@ class SystemRewriteDriverFactory : public RewriteDriverFactory {
     return 1;
   }
 
+  // Set the name of this process, for debugging visibility.
+  virtual void NameProcess(const char* name);
```

Comment:
It seems unfortunate to have these process-management methods in the factory class.  Can't we just have the factory generate a process-manager that will have these methods?